packages/react-menu/package.json

```diff
@@ -46,10 +46,10 @@
   },
   "dependencies": {
     "@fluentui/keyboard-key": "^0.2.13",
-    "@fluentui/react-make-styles": "^0.2.8",
-    "@fluentui/react-theme": "^0.3.3",
-    "@fluentui/react-theme-provider": "^1.0.0-beta.27",
-    "@fluentui/react-utilities": "^0.4.0",
+    "@fluentui/react-make-styles": "^9.0.0-alpha.0",
+    "@fluentui/react-theme": "^9.0.0-alpha.0",
+    "@fluentui/react-theme-provider": "^9.0.0-alpha.0",
```

Comment:
You mind doing me a solid and removing these dependencies if you're touching this file anyway ? 

```suggestion
```

Reply:
Awesome catch, will remove them 👍